src/components/StoriesCard/StoriesCard.vue

```diff
@@ -2,14 +2,27 @@
   <ion-card>
     <ion-card-header>
       <ion-card-title>Card Title</ion-card-title>
-      <ion-card-subtitle>Card Subtitle</ion-card-subtitle>
+      <ion-button v-if="!isElVisible" @click="doText(true)">Show</ion-button>
+      <ion-button v-else="isElVisible" @click="doText(false)">Hide</ion-button>
     </ion-card-header>
 
-    <ion-card-content>
+    <ion-card-content v-if="isElVisible">
       Here's a small text description for the card content. Nothing more,
       nothing less.
     </ion-card-content>
   </ion-card>
+
+  <ion-card>
+    <ion-card-header>
+      <ion-card-title>City Do-Do list</ion-card-title>
+    </ion-card-header>
+    <ion-card-header v-for="item in cityTask" :key="item.id">
+      <ion-card-title v-if="!item.status">{{ item.title }}</ion-card-title>
+      <ion-card-title v-else style="text-decoration: line-through solid !important;">{{ item.title }}</ion-card-title>
```

Comment:
Тут лучше сделать через динамичный класс и менять стили в `StoriesCard.scss`

```suggestion
      <ion-card class="stories-card">
        ...
        <ion-card-title class="stories-card__title" :class="{ 'is-done': item.isDone }">
          {{ item.title }}
        </ion-card-title>
        ...
      </ion-card>  
```

```scss
.stories-card {
  &.is-done {
    .stories-card__title {
      text-decoration: line-through solid !important;
    }
  }
}
```